starter/package.json

```diff
@@ -119,8 +119,8 @@
     "@angular/platform-server": "^16.2.12",
     "@angular/router": "^16.2.12",
     "@nationalbankbelgium/code-style": "^1.11.0",
-    "@nationalbankbelgium/stark-ui": "next",
-    "@nationalbankbelgium/stark-core": "next",
+    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-12.0.0-rc.0-f7a24d7d-1728638970.tgz",
```

Comment:
Please revert the changes the stark deps (should remain next/latest) in this file + adapt the engines also for the starter